data/plugins.json

```diff
@@ -157,23 +206,33 @@
     "safari": 10,
     "node": 6,
     "ios": 10,
+    "chrome-flag": null,
+    "edge-flag": 13,
+    "node-flag": 6,
     "opera": 37
   },
   "transform-exponentiation-operator": {
     "chrome": 52,
     "edge": 14,
     "firefox": 52,
+    "chrome-flag": 51,
+    "edge-flag": 14,
+    "node-flag": 6.5,
     "opera": 39
   },
   "transform-async-to-generator": {
     "chrome": 55,
     "firefox": 52,
+    "chrome-flag": 52,
+    "edge-flag": 13,
     "opera": 42
   },
   "syntax-trailing-function-commas": {
```

Comment:
This is allowing the user to add the trailing comma (babel-generator always removes trailing commas). Without the plugin it will error on unexpected token?

However, I just realized after your comment that we removed this restriction in babylon since its part of es2017 so I guess we don't really need this plugin anymore after a certain babylon version unless generator prints trailing commas by default later (no)